### PR TITLE
Luis

### PR DESCRIPTION
**Pull request by P4rz1val22**

### CODE REVIEW

File: src/clean-data.py [hunk 1]
```diff
@@ -66,34 +64,37 @@ def get_relative_month(text):
...
-        / df.loc[(df["Bedrooms"] != 0) & (df["Bathrooms"] != 0), "Bedrooms"]
+# # Compute the average number of bathrooms per bedroom, ignoring rows where bedrooms or bathrooms are zero
+# bathrooms_per_bedroom = (
+#     df.loc[(df["Bedrooms"] != 0) & (df["Bathrooms"] != 0), "Bathrooms"]
+#     / df.loc[(df["Bedrooms"] != 0) & (df["Bathrooms"] != 0), "Bedrooms"]
+# )
+# avg_bathrooms_per_bedroom = bathrooms_per_bedroom.mean()
+#
+# # Update the "Bathrooms" column where it is 0 and "Bedrooms" is not
+# df.loc[(df["Bathrooms"] == 0) & (df["Bedrooms"] != 0), "Bathrooms"] = np.round(
```
Comment (spencermendelsohn) [inline src/clean-data.py]: Remove unnecessary comments to clean up file

File: tests/housing-data-noAgencies.csv
Comment (spencermendelsohn) [inline tests/housing-data-noAgencies.csv:1]: Add this file to directory ***/Data***

File: tests/housing-data.csv
Comment (spencermendelsohn) [inline tests/housing-data.csv:1]: Same with this add it to the directory ***/Data***

File: tests/counting-agencies.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+lessThanAgencies = {}
+greaterThanAgencies = {}
+
+# Initialize a session using Amazon S3
+s3 = boto3.client("s3")
+
+# Specify the bucket name and the name you want to give the file
+bucket_name = "team-houses-bucket"
+key = "maggie-3.csv"
+name = "housing-data"
```
Comment (spencermendelsohn) [inline tests/counting-agencies.py]: Change these directory names after you moved them to `/Data`

**Review by spencermendelsohn — CHANGES_REQUESTED** [review]

File: tests/counting-agencies.py
Comment (spencermendelsohn) [inline tests/counting-agencies.py:1]: Recommend renaming to `tests/test-counting-agencies.py`

**Review by spencermendelsohn — CHANGES_REQUESTED** [review]

File: tests/counting-agencies.py
Comment (P4rz1val22) [reply tests/counting-agencies.py:1]: Moved to src cause it wasn't _really_ a test...

**Review by spencermendelsohn — APPROVED** [review]
SICK!!!

**Review by Grant-diane — APPROVED** [review]
yupp

**Pull request merged**